README.md

```diff
@@ -15,53 +15,30 @@ Build your own custom MLOps orchestration workflows from composable automation r
-
-What **FuseML** is **NOT**:
-
-* An opinionated open source MLOps platform. Flexibility and extensibility are FuseML's core principles. Instead of being a set of tightly integrated components, it relies on extension mechanisms and custom automation recipes to dynamically and loosely integrate the 3rd party AI/ML tools of your choice.
-* A complete lifecycle manager. FuseML's flexibility does come with a cost, which is vital to reduce complexity: managing the complete lifecycle (installation and upgrade processes) of supported 3rd party AI/ML tools is out of scope. However, FuseML will provide registration and auto-detection mechanisms for existing 3rd party tool installations, and may even go so far as to add lifecycle management to its list of supported extension mechanisms.
-
-## FuseML Principles
-
-* *Flexibility* - create and manage dynamic MLOps workflows connecting different AI/ML tools across multiple infrastructure domains
-
-* *Extensibility* - leverage FuseML's set of abstractions and extension mechanisms to add support for your favorite AI/ML tools
-
-* *Composability* - build complex MLOps workflows for your projects out of composable building blocks implementing a wide range of machine learning functions     
-
-* *Collaboration* - use MLOps automation and tool integration recipes created in collaboration by all AI/ML team roles - Data Scientists, Data Engineers, and DevOps Engineers
-
-* *GitOps for Machine Learning* - extend what traditional GitOps does with code to all other types of artifacts specific to machine learning - models and datasets, as well as other byproducts of a DevOps workflow (e.g. packages, container images) to provide features such a end-to-end versioning, tracking and reproducibility
-
-## Inception and Direction
-
-FuseML originated as a fork of our sister open source project [Carrier](https://github.com/SUSE/carrier), a lightweight open source PaaS built on top of Kubernetes, then has been gradually transformed and infused with the MLOps concepts that make it the AI/ML orchestration tool that it is today.
-
-The FuseML team is still in the process of defining the project strategy, but we can say with some certainty that the project will develop in the following main directions aligned with the stated principles:
+The project is under heavy development following the main directions:
 1. adding features and enhancements to improve flexibility and extensibility
-2. add support for more community shared AI/ML tools
-3. create more composable automation blocks adapted to the existing as well as new AI/ML tools
+2. adding support for more community shared AI/ML tools
+3. creating more composable automation blocks adapted to the existing as well as new AI/ML tools
+
+Take a look at [our Project Board](https://github.com/orgs/fuseml/projects/1) to see what we're working on and what's in store for the next release.
```

Comment:
With that it might make sense to remove line 9.

Reply:
good point